includes/model/class-application.php

```diff
@@ -87,7 +88,7 @@ public function get_id() {
 	 * @return string The User-Url.
 	 */
 	public function get_url() {
-		return $this->get_id();
+		return force_ssl( $this->get_id() );
```

Comment:
For things like `get_id()`, let's go as deep as we can to add the scheme, in this case `get_rest_url_by_path()` where `\get_rest_url()` also accepts a third argument for the scheme.